clients/cli/src/main/java/org/apache/gravitino/cli/commands/TagEntity.java

```diff
@@ -94,6 +97,9 @@ public void handle() {
       exitWithError(ErrorMessages.UNKNOWN_SCHEMA);
     } catch (NoSuchTableException err) {
       exitWithError(ErrorMessages.UNKNOWN_TABLE);
+    } catch (TagAlreadyAssociatedException tagAlreadyAssociatedException) {
+      exitWithError(
+          "[" + COMMA_JOINER.join(tags) + "]" + " are already associated with " + name.getName());
```

Comment:
This is not entirely correct, as it could be a subset of those tags. Does the exception include the tag name? I'd also drop the brackets.